torch/lib/c10d/Utils.hpp

```diff
@@ -19,6 +19,16 @@
 
 #include <c10d/Types.hpp>
 
+#ifdef _WIN32
+#include <winsock2.h>
+#include <ws2tcpip.h>
+#include <fcntl.h>
+#include <BaseTsd.h>
+#include <errno.h>
```

Comment:
This probably would end up unused. See the `EWOULDBLOCK` vs `WSAEWOULDBLOCK` part in https://github.com/pytorch/pytorch/pull/47749/files#r521358898

Reply:
Removed